Sources/SwiftSyntax/Raw/RawSyntax.swift

```diff
@@ -219,7 +219,7 @@ extension RawSyntax {
   /// If the syntax tree did not contain a token and thus no trivia could be attached to it, `nil` is returned.
   /// - Parameters:
   ///   - leadingTrivia: The trivia to attach.
```

Comment:
Not sure if you care about keeping parameter docstrings up to date, but if so this one’s missing.

Reply:
Thanks for noticing. Updated.